Examples/VaporExample/Package.swift

```diff
@@ -1,15 +1,15 @@
-// swift-tools-version:5.2
+// swift-tools-version:5.5
 import PackageDescription
 
 let package = Package(
     name: "VaporExample",
     platforms: [
-        .macOS(.v10_15)
+        .macOS(.v12)
```

Comment:
Vapor has not added support for concurrrency on old macOS versions (yet anyway, not sure if they plan to), so even if we did support it this would need to be 12+